intro/src/main/java/mate/academy/intro/dto/user/UserRegistrationRequestDto.java

```diff
@@ -0,0 +1,25 @@
+package mate.academy.intro.dto.user;
+
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.Size;
+import lombok.Data;
+
+@Data
+public class UserRegistrationRequestDto {
```

Comment:
I don't like solution where part of DTOs are records, and part are classes with @Data annotation.
This may cause a lot of frustration when you use next DTO and it again differs in behaviour from previous one.
Please unify and use either records or classes